pkg/azuredisk/azuredisk.go

```diff
@@ -98,6 +98,7 @@ const (
 	networkAccessPolicyField = "networkaccesspolicy"
 	diskAccessIDField        = "diskaccessid"
 	enableBurstingField      = "enablebursting"
+	csiMigrationField        = "csi-migration"
```

Comment:
Is this field added to the parameters passed to `CreateVolume` by the CSI Migration code? I couldn't find documentation on this.